Learn-JSON-Fetch-async-await-building-fcc-author-page/index.js

```diff
@@ -0,0 +1,45 @@
+const authorContainer = document.getElementById("author-container");
+const loadMoreBtn = document.getElementById("load-more-btn");
+let startingIndex = 0;
+let endingIndex = 8;
+let authorDataArr;
```

Comment:
```suggestion
let authorDataArr = [];
```

I'm personally not a fan of declaring without initialising (TypeScript would get angry that we're using `slice` on a value that might be `undefined`).